src/Compilers/CSharp/Portable/Compilation/MethodBodySemanticModel.cs

```diff
@@ -308,5 +308,12 @@ protected override void AnalyzeBoundNodeNullability(BoundNode boundRoot, Binder
         {
             NullableWalker.AnalyzeWithoutRewrite(Compilation, MemberSymbol, boundRoot, binder, diagnostics, createSnapshots);
         }
+
+        protected override bool IsNullableAnalysisEnabled()
+        {
+            return (MemberSymbol is SourceMemberMethodSymbol method) ?
+                Compilation.IsNullableAnalysisEnabledIn(method.SyntaxNode) :
```

Comment:
>Compilation.IsNullableAnalysisEnabledIn(method.SyntaxNode) [](start = 16, length = 58)

It doesn't look like this implements: "For constructors, the constructor is nullable-enabled if any of the containing type definitions overlap nullable-enabled contexts."